jdaviz/configs/specviz/plugins/unit_conversion/unit_conversion.vue

```diff
@@ -37,7 +37,7 @@
       ></v-text-field>
     </v-row>
 
-    <v-row>
+    <v-row v-if="spectral_axis_unit_equivalencies.length > 0">
```

Comment:
This is to avoid Unit Conversion plugin letting user convert unit only to fail right after.